### PR TITLE
refactor: move classes from components to button variant

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
As far as I can see, for each button in connections, we are using the same styles:
`ml-auto w-fit border-white px-3 font-medium text-white hover:bg-black`
Basically except for the ml-auto and w-fit most of it it's styling: border, font-size/color, hover color.
If it repeats that much, we should create a new variant and re-use it instead of typing it each time.

## Linear Ticket
https://linear.app/autokitteh/issue/UI-710/create-variant-for-the-button-in-connections

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [x] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/components/molecules/menu/menu.tsx
Comment (J1za) [inline src/components/molecules/menu/menu.tsx:1]: white button text
image
Logo not aligned 
image

**Comment by RonenMars** [comment]
Fixed the design, last 2 commits

**Comment by J1za** [comment]
white text 
image

**Comment by RonenMars** [comment]
@J1za Fixed bottom sidebar buttons

**Comment by J1za** [comment]
image
not align, and when I open the sidebar I see the sidebar logo moving but it shouldn't

**Pull request closed without merging**